packages/fxa-payments-server/.storybook/styles.scss

```diff
@@ -1,5 +0,0 @@
-// HACK: <div id="root"> is a part of the Storybook template, try to minimize
```

Comment:
I removed this file because we don't have it in any other Storybooks and I'm not sure why we need it. Happy to add it back if there's a reason to.